docs/DeveloperGuide.md

```diff
@@ -155,6 +155,70 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Create Doctor/ Patient feature
+
+##  Create Doctor feature
+
+The CreateDoctorCommand is implemented and used to add a new doctor to the address book.
```

Comment:
should it be add a new doctor to `MedDict database which is the address book` instead of `address book` only? This issue appears in line 185, 203, 212 etc. Consider to change the words address book or use another expression for it to fit our MedDict product.